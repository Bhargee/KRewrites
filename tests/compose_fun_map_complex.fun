"foldl" (fun x -> fun y -> x + y) ("foldl" (fun z -> fun w -> z - w) ("map" (fun
a -> a + 2)))
