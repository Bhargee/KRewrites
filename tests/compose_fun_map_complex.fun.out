<T>
    <k>
        ("foldl" (fun (x -> (fun (y -> (x + y)))))) ("foldl" (fun (
          #symExp(1) -> ((fun (z -> (fun (w -> (z - w))))) ((fun (a -> (a + 2
          ))) #symExp(1))))))
    </k>
    <store>
        .Map
    </store>
    <env>
        .Map
    </env>
</T>
